packages/api/src/objectSet/EXPERIMENTAL_ObjectSetListener.ts

```diff
@@ -25,6 +25,12 @@ export interface EXPERIMENTAL_ObjectSetListener<
    */
   onChange?: (objects: Array<OsdkObjectOrInterfaceFrom<O>>) => void;
 
+  /**
```

Comment:
If we're aiming to get this out to some nucleus folks, will delete this and also add information on add/delete that we aren't using right now